src/python/pants/notes/2.2.x.md

```diff
@@ -9,6 +9,24 @@ https://raw.githubusercontent.com/pantsbuild/setup/2f079cbe4fc6a1d9d87decba51f19
 This release also removes the ability to use `%(pants_bootstrapdir)s` and `%(pants_configdir)s` in
 interpolations in your config file.
 
+## 2.2.0.dev2 (Dec 16, 2020)
+
+### Plugin API Changes
+
+* Upgrade packaging from 20.4 to 20.7 ([#11302](https://github.com/pantsbuild/pants/pull/11302))
+
+* Update default values for run tracker and streaming work unit handlers to more commonly used values. ([#11292](https://github.com/pantsbuild/pants/pull/11292))
+
+### Bug fixes
+
+* Fix filtering of log messages generated in native code. ([#11313](https://github.com/pantsbuild/pants/pull/11313))
+
+* Upgrade to Pex 2.1.24. ([#11312](https://github.com/pantsbuild/pants/pull/11312))
```

Comment:
Bump on mentioning Big Sur

Reply:
Argh. It's apparently quite hard to move a pull request from being opened from origin to being opened from a branch. Thought I had, and that I'd force pushed this fix. Will make it shortly before landing.